en/firefox/features/password-manager.ftl

```diff
@@ -0,0 +1,43 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/features/password-manager
+
+# HTML page title
+password-manager-firefox-browser = { -brand-name-firefox-browser }: Fast, Easy Password Manager
+
+# HTML page description
+password-manager-firefox-password = { -brand-name-firefox } Password Manager saves all your passwords in one place so you can automatically login to sites, or retrieve saved passwords.
+
+# Hero title
+password-manager-password-manager = Password Manager achievement unlocked
+password-manager-give-up-the-memory = Give up the memory game with { -brand-name-firefox } Password Manager.
+password-manager-password-hero = Password hero
+
+# Variables:
+#   $url (url) = link to https://support.mozilla.org/kb/use-master-password-protect-stored-logins
+password-manager-forget-the-reset = Forget the reset. { -brand-name-firefox } Password Manager keeps all your passwords so you can log in automatically, or find saved passwords easily. For super security, give your computer a <a href="{ $url }">master password</a>.
+
+password-manager-password-magician = Password magician
+
+# Obsolete string
+# Meaning its a quick and competent password manager
+password-manager-password-ninja = Password ninja
+
+password-manager-no-more-try-again = No more “try again” while you’re trying to get somewhere. Log in to your { -brand-name-firefox-account } on your phone, and your passwords come with you. Your login details will simply appear, just like that.
+password-manager-password-champion = Password champion
+
+# Variables:
+#   $addons (url) = https://addons.mozilla.org/firefox/collections/mozilla/password-managers/
+password-manager-earn-your-security-gold-medal = Earn your security gold medal with { -brand-name-firefox }’s vast array of password manager <a href="{ $addons }">add-ons</a>. Choose an existing favorite or find a next-level one through expert community ratings and reviews.
+
+# Obsolete string
+# A way of saying this password manager is really good at what is does master here = mastery of subject
```

Comment:
Though this is a note, not impacting translation, `what is does` should be `what it does`

Reply:
Updated!